tree/treeplayer/src/TDFInterfaceUtils.cxx

```diff
@@ -153,13 +153,30 @@ SelectColumns(unsigned int nRequiredNames, const ColumnNames_t &names, const Col
    }
 }
 
+bool IsTreeLeaf(TTree &t, const std::string &leaf)
+{
+   // TODO understand why GetBranch is also needed (run the tests without, inspect failures)
```

Comment:
There is at least the odd case:  ```tree->Branch("brname","leafname/I");``` where we probably want to allow plotting with either ```brname``` or ```brname.leafname```

Reply:
Do you mean a branch with an individual leaf? Should we really want to support that in the high level interface?

Reply:
Well ... I was speculating on the possible source of the problem that was mentioned.
In addition, yes I think we ought to let the user write either ```brname``` or ```brname.leafname``` (and I thought it was already the case in TDF ... it is the case in TTree::Draw).